src/model/Fiends.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
```

Comment:
imo it would be good to namespace everything to prevent any chance of collisions with other libraries (or in the unlikely event someone used this a dependency for their project).